datafusion/functions/benches/contains.rs

```diff
@@ -0,0 +1,162 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Would it be possible to break this benchmark into its own PR?

This is mostly a selfish ask as I have comparison scripts for performance that compare performance against main, and they only work if the benchmark is on main as well